File: src/level3/services.js
// Level 3: Service layer
const { level4Database } = require('../level4/database');

function level3Service() {
  return level4Database() + ' -> L3:service';
}

function level3ApiCall() {
  return 'L3:apiCall';
}

module.exports = {
  level3Service,
  level3ApiCall
};

File: src/py_level3/services.py
"""Level 3: Services module"""
import sys
import os
sys.path.insert(0, os.path.join(os.path.dirname(__file__), '..'))

from py_level4.database import level4_database

def level3_service():
    return level4_database() + " -> L3:service"

def level3_api_call():
    return "L3:api_call"
